src/rqt_console/message_data_model.py

```diff
@@ -121,7 +124,11 @@ def data(self, index, role=None):
                     else:
                         data = getattr(msg, column)
                     # <font> tag enables word wrap by forcing rich text
-                    return '<font>' + data.decode('utf-8') + '<br/><br/>' + \
+                    # Python version is checked here in order to solve 
+                    # https://github.com/ros-visualization/rqt_console/issues/33
+                    # Python 2 requres the data to be decoded to avoid crash when the string contains
```

Comment:
Great, thanks for the comment I asked for.
Please just correct the typo in requires and I can merge your PR. :)

Reply:
Apologies for the delay, I fixed the typo